producers-commissioning/src/main/java/uk/gov/moj/sdt/producers/comx/config/bulkfeedback/BulkFeedbackConfigCOne.java

```diff
@@ -0,0 +1,689 @@
+package uk.gov.moj.sdt.producers.comx.config.bulkfeedback;
+
+import java.time.LocalDateTime;
+
+import com.google.common.collect.Lists;
+import org.springframework.beans.factory.config.MethodInvokingFactoryBean;
+import org.springframework.boot.autoconfigure.EnableAutoConfiguration;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import uk.gov.moj.sdt.domain.BulkSubmission;
+import uk.gov.moj.sdt.producers.comx.utils.BulkFeedbackFactory;
+
+@Configuration
+@EnableAutoConfiguration
+public class BulkFeedbackConfigCOne {
+
+    @Bean("uk.gov.moj.sdt.producers.comx.utils.BulkFeedbackFactoryC00000001")
+    public BulkFeedbackFactory bulkFeedbackFactory() {
+        return new BulkFeedbackFactory(bulkSumissionC00000001());
+    }
+
+    private BulkSubmission bulkSumissionC00000001() {
+        BulkSubmission bulkSubmission = new BulkSubmission();
+        bulkSubmission.setCustomerReference("USER_FILE_REFERENCE_A1");
+        bulkSubmission.setNumberOfRequest(106);
+        bulkSubmission.setSubmissionStatus("Completed");
+        bulkSubmission.setCreatedDate(LocalDateTime.of(2013, 7, 22, 13, 0));
```

Comment:
Parameters should be 2014, 1, 22, 13, 0.

Reply:
done